crates/viewer/re_global_context/src/lib.rs

```diff
@@ -71,7 +71,7 @@ pub enum DisplayMode {
     LocalTable(TableId),
 
     /// The Redap server/catalog/collection browser.
-    RedapEntry(re_log_types::EntryId),
```

Comment:
If you haven't done so, you might want to grep for `RedapEntry(entry_id)` -> `RedapEntry(entry_uri)`, for the sake of reducing confusion.